www/include/configuration/configGenerate/common-Func.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+/*
+ * Copyright 2005-2022 Centreon
+ * Centreon is developped by : Julien Mathis and Romain Le Merlus under
+ * GPL Licence 2.0.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation ; either version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, see <http://www.gnu.org/licenses>.
+ *
+ * Linking this program statically or dynamically with other modules is making a
+ * combined work based on this program. Thus, the terms and conditions of the GNU
+ * General Public License cover the whole combination.
+ *
+ * As a special exception, the copyright holders of this program give Centreon
+ * permission to link this program with independent modules to produce an executable,
+ * regardless of the license terms of these independent modules, and to copy and
+ * distribute the resulting executable under terms of Centreon choice, provided that
+ * Centreon also meet, for each linked independent module, the terms  and conditions
+ * of the license of that module. An independent module is a module which is not
+ * derived from this program. If you modify this program, you may extend this
+ * exception to your version of the program, but you are not obliged to do so. If you
+ * do not wish to do so, delete this exception statement from your version.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+/**
+ * format and add debug in CentreonXML
+ *
+ * @param CentreonXML $xml
+ * @param mixed[] $tabs
+ * @return int
+ */
+function printDebug($xml, array $tabs): int
+{
+    global $pearDB, $ret, $centreon, $nagiosCFGPath;
+
+    $DBRESULT_Servers = $pearDB->query(
+        "SELECT `nagios_bin` FROM `nagios_server`
+        WHERE `localhost` = '1' ORDER BY ns_activate DESC LIMIT 1"
+    );
+    $nagios_bin = $DBRESULT_Servers->fetch();
```

Comment:
```suggestion
    $nagios_bin = $DBRESULT_Servers->fetch();
```
I know it's a copy paste but could you handle variables in camelCase :pray: